Libraries/Core/NativeExceptionsManager.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+import type {StackFrame} from './Devtools/parseErrorStack';
+
+export interface Spec extends TurboModule {
+  +reportFatalException: (
+    message: string,
+    stack: Array<StackFrame>,
+    exceptionId: number,
+  ) => void;
+  +reportSoftException: (
+    message: string,
+    stack: Array<StackFrame>,
+    exceptionId: number,
+  ) => void;
+  +updateExceptionMessage: (
+    message: string,
+    stack: Array<StackFrame>,
+    exceptionId: number,
+  ) => void;
+  /**
+   * Available only on Android
+   */
+  +dismissRedbox: () => void;
```

Comment:
what do we do about it @rickhanlonii?

Reply:
Should we need to split it to Android and iOS? if not, seems we need to label it for Codegen.

Reply:
For additional properties we have decided to generate it on both platforms and no-op on one, so for this you can update the comment to `// Android only`

Splitting out would be required for props/events that conflict